pkgs/tools/package-management/nix/common.nix

```diff
@@ -246,6 +250,11 @@ self = stdenv.mkDerivation {
   # See https://github.com/NixOS/nix/issues/5687
   + lib.optionalString (atLeast25 && stdenv.isDarwin) ''
     echo "exit 99" > tests/gc-non-blocking.sh
+  '' + ''
+    # For some reason when building nixStatic (some form of cross-compilation),
+    # PATH doesn't get extended correctly by installCheckInputs
```

Comment:
Also related to `native*Inputs` I think. The regular ones aren't added to PATH afaik.

Reply:
Indeed it does.

Reply:
The comment does not relate to the following line any more.